### ISSUE TITLE
Documentation for commit hook is very confusing

### ISSUE THREAD

**Issue by ssokolow** [landscapeio/landscape-issues#14]
To be perfectly honest, at present, there is little the documentation for the commit hook could do to be more confusing while still being on-topic.
1. The instructions say nothing about "Hook Url" and I had to google up the pull request that added the hook to GitHub before I realized it was optional.
2. The following section of "Install Notes" is currently flat-out wrong and I wasted 15 minutes trying to find said control panel on Landscape before I gave up and followed the "If you wish to install the hook manually" line.

> If you have created an account already at https://landscape.io, you can simply enable or disable checking of your repositories. The hooks will automatically be installed and uninstalled.

I suggest the following fixes:
1. Hyperlink "Landscape" at the beginning of the first sentence so discoverability of the site via the GitHub hooks list won't drop when the rest of this list is implemented.
2. Change the "Hook Url" field label to "Hook URL (optional)"
3. Remove the "If you have created an account" text and rephrase the "If you wish to install the hook manually" text to acknowledge that, at the moment, manual is the only option.
4. Remove the explicit word-wrapping in the source file so there are no spurious line-breaks making it harder to identify where the paragraphs begin and end in the rendered result.

**Comment by carlio** [comment]
Ok, I finally got around to addressing this.

A little back story: the intention was for hooks to be automatically managed by Landscape, however that requires write permissions for the repository, and at the moment I don't see a huge need to ask for that, as some people are, quite reasonably, unwilling to allow permissions for a service like Landscape which really only needs read-only permissions. Until there is better granularity in the scopes of the GitHub API, I'll probably keep it this way. As a result, Landscape cannot automatically manage hooks.

Once I gave up on the hook management, I forgot to think about how the hook would appear in the GitHub service hook list. It now has correct documentation, and I have ditched the unecessary configuration options for now. It is much easier to enable, and the documentation is correct.

This does require a change from the GitHub guys (see the pull request in the comment above this one), and once that ends up merged, I'll close this issue.

Cheers!
